operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -66,6 +66,14 @@ kiali_defaults:
       custom_metrics_url: ""
       url: ""
     tracing:
+      auth:
```

Comment:
don't forget to also update the kiali_cr.yaml file which stands as a documentation

Reply:
oh you're right

Reply:
@jotak  Did you try this ? I was checking and I didn't need to set the url https://jaeger-query.istio-system.svc to fetch the traces o.O and it's works...I only see this if jaeger is in another namespace

Reply:
For example for jaeger-query would be https://jaeger-query.istio-system.svc:16686 and health check  https://jaeger-query.istio-system.svc:16687